xml/depl_maintenance.xml

```diff
@@ -312,6 +312,175 @@
+     most likely reason is a problem with STONITH devices.
+    </para>
+   </step>
+   <step>
+    <para>
+     Determine the cause of the &compnode;'s failure.
+    </para>
+   </step>
+   <step>
+    <para>
+     Boot the &compnode; again.
+    </para>
+   </step>
+   <step>
+    <para>
+     Check whether the <systemitem>crowbar_join</systemitem> script ran
+     successfully on the &compnode;. If this is not the case, check the log
+     files to find out the reason.
+    </para>
+   </step>
+   <step>
+    <para>
+     If the <systemitem>chef-client</systemitem> agent triggered by
+     <systemitem>crowbar_join</systemitem> succeeded, confirm that the
+     <systemitem>pacemaker_remote</systemitem> service is up and
+     running.
+    </para>
+   </step>
+   <step>
+    <para>
+     Check whether the remote node is registered and considered healthy by the
+     core cluster. If this is not the case check
+     <filename>/var/log/pacemaker.log</filename> on the Designated Co-ordinator
+     to determine the cause. There should be a remote primitive running on the
+     core cluster (active/passive). This primitive is responsible for
+     establishing a TCP connection to the
+     <systemitem>pacemaker_remote</systemitem> service on the &compnode;. Use
+     standard network debugging methods to make sure that nothing is preventing
+     this particular TCP connection from being established (for example, problems with
+     NICs, switches, firewalls etc.).
+    </para>
+   </step>
+   <step>
+    <para>
+     If Pacemaker can communicate with the remote node, it starts the
+     <systemitem>nova-compute</systemitem> service on it as part of the cloned
+     group <literal>cl-g-nova-compute</literal> using the NovaCompute OCF
+     resource agent. This cloned group will block startup of
+     <systemitem>nova-evacuate</systemitem> until at least one clone is
+     started. Therefore the next section is necessary as a related but
+     different procedure. See also the next step.
+    </para>
+   </step>
+   <step>
+    <para>
+     It may happen that <systemitem>NovaCompute</systemitem> has been launched
+     correctly on the &compnode; by <systemitem>lrmd</systemitem>, but the
+     <systemitem>openstack-nova-compute</systemitem> service is still not
+     running. This usually happens when <systemitem>nova-evacuate</systemitem>
+     did not run correctly. If <systemitem>nova-evacuate</systemitem> is not
+     running on one of the core cluster nodes, make sure that the service is
+     marked as started (<literal>target-role="Started"</literal>). If this is
+     the case, then your cloud probably does not meet the assumption of this
+     procedure. In case <systemitem>nova-evacuate</systemitem> is started but
+     it is failing, check the Pacemaker logs to determine the cause. When
+     <systemitem>nova-evacuate</systemitem> is started and functioning
+     correctly, it should call nova's evacuate API to release resources
```

Comment:
Let's put "evacuate" in quotes here to emphasise that it isn't really about evacuation.

Reply:
@dmpop, I think *\<literal>evacuate\</literal> API* would make sense, because it is an API name. Alternatively, you could call it *evacuation API* which would be proper English but not be a proper reproduction of the actual API name.